handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -248,6 +249,14 @@ protected void deallocate() {
         }
     }
 
+    /**
+     * Enables (or disables) OCSP stapling on the {@link SSLEngine}.
+     */
+    @UnstableApi
+    public synchronized boolean enableOcsp(boolean enabled) {
+        return SSL.setTlsExtStatusType(ssl, enabled ? SSL.TLSEXT_STATUSTYPE_ocsp : 0);
```

Comment:
Ack. Happy to change it into `enableOcsp()` as there is no use-case for the toggle functionality.